test/multi_domain.jl

```diff
@@ -0,0 +1,215 @@
+using ModelingToolkitStandardLibrary.Electrical
+using ModelingToolkitStandardLibrary.Mechanical.Rotational
+using ModelingToolkitStandardLibrary.Blocks
+using ModelingToolkitStandardLibrary.Blocks
+using ModelingToolkitStandardLibrary.Thermal
+import ModelingToolkitStandardLibrary
+using ModelingToolkit, OrdinaryDiffEq, Test
+# using Plots
+
+@parameters t
+D = Differential(t)
+
+@testset "DC motor" begin
+    R = 0.5
+    L = 4.5e-3
+    k = 0.5
+    J = 0.02
+    f = 0.01
+    V_step = 10
+    tau_L_step = -3
+    @named ground = Ground()
+    @named source = Voltage()
+    @named voltage_step = Blocks.Step(height = V_step, start_time = 0)
+    @named R1 = Resistor(R = R)
+    @named L1 = Inductor(L = L)
+    @named emf = EMF(k = k)
+    @named fixed = Fixed()
+    @named load = Torque(use_support = false)
+    @named load_step = Blocks.Step(height = tau_L_step, start_time = 3)
+    @named inertia = Inertia(J = J)
+    @named friction = Damper(d = f)
+
+    connections = [connect(fixed.flange, emf.support, friction.flange_b)
+                   connect(emf.flange, friction.flange_a, inertia.flange_a)
+                   connect(inertia.flange_b, load.flange)
+                   connect(load_step.output, load.tau)
+                   connect(voltage_step.output, source.V)
+                   connect(source.p, R1.p)
+                   connect(R1.n, L1.p)
+                   connect(L1.n, emf.p)
+                   connect(emf.n, source.n, ground.g)]
+
+    @named model = ODESystem(connections, t,
+                             systems = [
+                                 ground,
+                                 voltage_step,
+                                 source,
+                                 R1,
+                                 L1,
+                                 emf,
+                                 fixed,
+                                 load,
+                                 load_step,
+                                 inertia,
+                                 friction,
+                             ])
+    sys = structural_simplify(model)
+
+    @test_broken prob = ODAEProblem(sys, Pair[], (0, 6.0))
```

Comment:
What does this fail with? Have you tried ODEProblem?

Reply:
The `ODAEProbem` construction works fine but solve errors. `ODEProblem` works. https://github.com/SciML/ModelingToolkit.jl/issues/1867

Reply:
Could you add a test with ODEProblem?